src/main/Restaurant.java

```diff
@@ -0,0 +1,19 @@
+package main;
+
+public class Restaurant {
+    private float cash;
+
+    //Constructor:
+    public Restaurant() {
+        this.cash = 100000;
+    }
+
+    //getter and setter: cash
+    public float getCash() {
+        return cash;
+    }
+
+    public void setCash(float cash) {
+        this.cash = cash;
```

Comment:
I would prefer that the value be added to the current balance

Reply:
I'm working on that. Let me a moment